demo/src/main/java/com/orange/ods/demo/ui/components/cards/CardHorizontal.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.demo.ui.components.cards
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import coil.compose.rememberAsyncImagePainter
+import com.orange.ods.compose.component.card.OdsHorizontalCard
+import com.orange.ods.demo.R
+import com.orange.ods.demo.domain.recipes.LocalRecipes
+import com.orange.ods.demo.ui.components.utilities.clickOnElement
+
+@Composable
+fun CardHorizontal(customizationState: CardCustomizationState) {
+    val context = LocalContext.current
+    val recipes = LocalRecipes.current
+    val recipe = rememberSaveable { recipes.filter { it.description.isNotBlank() }.random() }
+
+    with(customizationState) {
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(dimensionResource(id = R.dimen.spacing_m))
+                .verticalScroll(state = rememberScrollState()),
+            verticalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.spacing_m))
+        ) {
+            val button1Text = stringResource(id = R.string.component_element_button1)
+            val button2Text = stringResource(id = R.string.component_element_button2)
+            val cardContainerText = stringResource(id = R.string.component_card_element_container)
```

Comment:
String value for `component_card_element_container` is `Card container`. Maybe we could rename it to `card` ?